src/GSoft.Extensions.Http.Authentication.ClientCredentialsGrant/ClientCredentialsOptions.cs

```diff
@@ -32,12 +32,17 @@ public sealed class ClientCredentialsOptions
     /// </summary>
     public string Scope
     {
-        get => string.Join(" ", this.Scopes);
+        get => this.Scopes == null ? string.Empty : string.Join(" ", this.Scopes);
```

Comment:
Would it be better to use the `string.IsNullOrWhitespace` comparison?

Reply:
`this.Scopes` is an array of string si this doesn't apply

Reply:
Good point!